airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/activities/GenerateInputActivityTest.java

```diff
@@ -225,6 +239,8 @@ void testGetResetSyncWorkflowInput() throws IOException, ApiException {
         .withDestinationConfiguration(DESTINATION_CONFIGURATION)
         .withState(STATE);
 
+    verify(oAuthConfigSupplier).injectDestinationOAuthParameters(DESTINATION_DEFINITION_ID, WORKSPACE_ID, DESTINATION_CONFIGURATION);
```

Comment:
nit: could this test ensure that a secret was decrypted?

Reply:
Updated the test to actually check that we're using the config returned by the injection method. Internals of how the injection works left up to unit tests for the OAuthConfigSupplier.